lib/std/crypto/pcurves/p256.zig

```diff
@@ -49,21 +49,26 @@ pub const P256 = struct {
     }
 
     /// Create a point from affine coordinates after checking that they match the curve equation.
-    pub fn fromAffineCoordinates(x: Fe, y: Fe) EncodingError!P256 {
+    pub fn fromAffineCoordinates(p: AffineCoordinates) EncodingError!P256 {
```

Comment:
FYI
this should not really be considered breaking because `fromAffineCoordinates` was introduced a week ago